examples/context/motivation-problem.js

```diff
@@ -0,0 +1,34 @@
+class Button extends React.Component {
+  render() {
+    return (
+      <button style={{background: this.props.color}}>
+        {this.props.children}
+      </button>
+    );
+  }
+}
+
+class Message extends React.Component {
+  render() {
+    // highlight-range{1-3}
+    // The Message component must take `color` as as prop to pass it
+    // to the Button. Using context, the Button could connect to the
+    // color context on its own.
```

Comment:
I got stuck reading this example for 10 seconds looking for usage of context. The second sentence refers to context as if *we're already using it here*. I find this confusing.

I would prefer if it described *how it works in this example*, and explained the pain point. And the comment in the solution would describe the solution and why it solves the pain point.